Postman Test Suite/Ed-Fi ODS-API Integration Test Suite ResponseTests.postman_collection.json

```diff
@@ -33,12 +33,12 @@
 									"pm.test(\"Should include a server-generated correlation id in the error response\", () => {\r",
 									"  const problemDetails = pm.response.json();\r",
 									"\r",
-									"  pm.expect(problemDetails.correlationId).to.exist;\r",
-									"\r",
-									"  pm.expect(problemDetails.errors[0]).to.equal(\"Unexpected character encountered while parsing value: }. Path '', line 0, position 0.\");\r",
+									"  // Postman tests provide a 12-character correlation id. Make sure that is not the one provided in the response.\r",
```

Comment:
This test was actually not testing the behavior because the collection level logic was still adding it. The expectation is now written to prevent that situation from happening again by ensuring it's not 12 characters in length (indicating that the API is responsible for the correlation id rather than the test collection's logic).